src/hooks/useTasks.js

```diff
@@ -35,10 +37,11 @@ function useOpenTasks(tasks, now, courtSettings) {
     if (!convertedTasks) {
       return []
     }
-
     const openTasks = []
     const incompleteTasks = convertedTasks.filter(
```

Comment:
Looking at this again, what do you think about lifting `incompleteTasks` into its own `useMemo()`, and then just using that directly in this later `useMemo()`?

Reply:
Could you explain why that would be better? since  that new useMemo will also have as a dependency the `convertedTasksPhasesKey` like the the memo where it is right now